packages/yarnpkg-builder/package.json

```
@@ -13,7 +13,7 @@
     "babel-loader": "^8.1.0",
     "chalk": "^3.0.0",
     "clipanion": "^3.0.0-rc.10",
-    "esbuild-wasm": "^0.8.36",
+    "esbuild-wasm": "^0.10.0",
```

Hey while you're here, would you mind fixing the `esbuild` peer dependency for `@yarnpkg/builder`? 😄

Currently it triggers a warning about an unmet peer dependency during installation, because all the other esbuild dependencies are of the form: `"esbuild": "npm:esbuild-wasm@..."`; except for this one.

Would you consider fixing it as part of this change, or would you rather it be separate?

```suggestion
    "esbuild": "npm:esbuild-wasm@^0.10.0",
```

Just to point out the obvious, it would also require a change to:
`packages/yarnpkg-builder/sources/commands/build/bundle.ts:8`

From:

```typescript
import {build, Plugin}                                                      from 'esbuild-wasm';
```

To: 

```typescript
import {build, Plugin}                                                      from 'esbuild';
```

---

Sure 👍